File: Evaluacion1/app/src/main/java/com/example/evaluacion1/MainActivity.java
package com.example.evaluacion1;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void Verdatos(View view)
    {
        startActivity(new Intent(MainActivity.this, Datos.class));
    }
    public void regitrar(View view)
    {
        startActivity(new Intent(MainActivity.this, Registrar.class));
    }
    public void ver(View view)
    {
        startActivity(new Intent(MainActivity.this, Ver.class));
    }
}

File: AppRegistro Kotlin/AppRegistro/app/src/main/java/com/example/appregistro/MainActivity.kt
package com.example.appregistro

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView
import com.example.appregistro.Data.Data
import com.google.android.material.floatingactionbutton.FloatingActionButton

class MainActivity : AppCompatActivity() {

    lateinit var txtLista:TextView
    lateinit var btnNuevo:FloatingActionButton

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        txtLista=findViewById(R.id.txtListado)
        btnNuevo=findViewById(R.id.btnNuevo)

        //abrir vista para nuevo registro
        btnNuevo.setOnClickListener(){
            intent= Intent(this.applicationContext,RegistroActivity::class.java)
            startActivity(intent)
        }

        //cargar datos del listado
        if(Data.lista_personas.size>0){
            var texto:String=""
            for (item in Data.lista_personas){

                texto=texto+ "ID:${item.id} Nombre:${item.nombre}  Direccion:${item.direccion}  \n\n"
            }
            txtLista.text=texto
        }else
            txtLista.text="No existen registros..."

    }
}